lib/meson.build

```diff
@@ -1,56 +1,85 @@
 pkgconfig = import('pkgconfig')
 
-sources = files(
-  'vkvideodecodeparser.cpp',
-  'gstvkh264dec.cpp',
-  'gstvkh265dec.cpp',
-  'gstvkvideoparser.cpp',
-  'videoutils.c',
+codecparser_sources = files (
```

Comment:
isn't possible to bundle gstharness with gstvkvideoparser avoiding another static library?

Reply:
doh, it is bundled... didn't read it right

Reply:
I wanted to stay the closest from the orginal code but just setting the testclock to NULL and not invoke `GST_TEST_CLOCK_CAST (gst_test_clock_new ());. `